CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+## Development setup
+We welcome any quality bugfixes or contributions!
+
+To avoid a conflicts with your existing installation, delete the installed extension at `~/.vscode/extensions/vsmobile.vscode-react-native`.
+
+### Windows
+* In `C:/Users/<username>/.vscode/extensions/`, `git clone` this repository
+
+### macOS/Linux
+* `git clone` this repository
+* Run `ln -s <path to repo> ~/.vscode/extensions/vscode-react-native`
+* You could clone it to the extensions directory if you want, but working with hidden folders in macOS can be a pain.
+
+### Then...
+* `cd` to the folder you just cloned
+* Run `npm install -g gulp` and `npm install`
+    * You may see an error if `bufferutil` or `utf-8-validate` fail to build. These native modules required by `ws` are optional and the debug adapter should work fine without them.
+* Run `gulp build`
+
+## Debugging
+There are currently 3 components to our extension: The extension running in the VS Code process, the debug adapter, and some code wrapping the user react-native code which is launched by the debug adapter. These are all debugged in different ways:
+
+* To debug the extension process itself, in VS Code run the `Launch Extension` debug target which will spawn a new instance of VS Code with the extension installed. You can set breakpoints in the TypeScript and debug things such as extension activation and the command palette.
```

Comment:
I'm not sure if what I did here was correct. I was having trouble determining when it's react-native vs. React Native.